src/System.AspNetCore/Microsoft.Omex.System.AspNetCore.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

Comment:
Shouldn't we use the directory.build.props variables instead: <TargetFrameworks>$(TargetFrameworks);$(NetStandardVersion)</TargetFrameworks>?

Reply:
The classes in AspNetCore require version >= net461; should I create another variable and use it?

Reply:
Not sure about the expected setup here. @AndreyTretyak should have a better idea.

Reply:
Should be fine for now, since we have required functionality only starting from net461, in future, we  may need to create a property for it